packages/kyodo-contracts/contracts/MembersV2.sol

```diff
@@ -10,12 +10,23 @@ contract MembersV2 is MembersV1 {
     external
   {
     require(nickNameNotExist(_value));
+    
+    bool isAliasEmpty = _value.toSlice().empty();
+
     string storage prevValue = whitelist[msg.sender].alias;
+
+    if (isAliasEmpty) {
+      require(!prevValue.toSlice().empty());
+    }
+
     int prevIndex = getNickNameIndex(prevValue);
+
     whitelist[msg.sender].alias = _value;
 
     // update used aliases
-    usedAliases.push(_value);
+    if (!isAliasEmpty) {
```

Comment:
This check is obsolete as the `require(!prevValue.toSlice().empty());` block above would terminate the execution

Reply:
Require block would terminate only in case alias is empty and previous value was empty

Reply:
You're right!